docs/modules/reference/pages/billing/subscription/user-subscription.adoc

```diff
@@ -0,0 +1,119 @@
+= Manage Users
+
+Payara Cloud allows you to have multiple users accessing your subscription.
+In order to do so you invite the user into subscription via email.
+Subsequently, they would register or use their existing Payara Cloud account and get access to the subscription.
+
+== Roles
+
+We recognize multiple roles of users in Payara Cloud:
+
+Billing Manager::
+Users in this role have access to all billing and payment information related to the subscription.
+They can also invite new users to subscription and alter their roles.
+They do not necessarily have permissions to deploy applications deployed to Payara Cloud.
+
+Administrator::
+Administrators have full permissions to work deploy and configure applications deployed to Payara Cloud.
+They can also invite new user to subscription and alter their roles.
+
+Billing Manager with Administrative Privilege::
+This role is union of two roles above, granting you all the permissions.
+When you sign up to Payara Cloud, this is the role you have in your subscription.
+
+Member::
+Member has restricted access to subscription's applications. Namely, they cannot create, delete or modify namespaces.
+
+=== Role Requirements
```

Comment:
I assume this rule is automatically satisfied when I sign up new because the first signup account always has both roles. Going by this, does it mean the first signup account can have the roles removed? I expect that account is the superuser?

Reply:
If you have other users to satisfy billing manager and administrator roles, the initial account can indeed get removed. There's no greater power in the universe than Billing Manager with Admin priv. (ok, maybe postgresql admin account is greater ;))